app/css/animator.css

```diff
@@ -505,4 +499,91 @@ a {
 
 ::-webkit-scrollbar-corner       { background-color: #000; }
 
-::-webkit-resizer                { }
\ No newline at end of file
+::-webkit-resizer                { }
+
+/* ======== Preference Windows ======== */
```

Comment:
Please move all this (Preference Windows???, Projects Area, and Recent News to `index.css`. You will also need to copy the scrollbar styling into it.